src/schedulers/VirtualTimeScheduler.ts

```diff
@@ -14,7 +14,7 @@ export default class VirtualTimeScheduler implements Scheduler {
   protected static frameTimeFactor: number = 10;
   
   now() {
-    return this.frame * VirtualTimeScheduler.frameTimeFactor;
+    return this.frame;
   }
```

Comment:
This one was actually wrong since frame is already multiplied by frameTimeFactor inside the TestScheduler parser of marbles.